Analytics/Integrations/Taplytics/SEGTaplyticsIntegration.m

```diff
@@ -28,9 +28,17 @@ - (id)init {
 }
 
 - (void)start {
-  [Taplytics startTaplyticsAPIKey:[self apiKey]];
+  NSDictionary *options = [[NSMutableDictionary alloc] init];
   
-  SEGLog(@"TaplyticsIntegration initialized with api key %@", [self apiKey]);
+  if ([self delayLoad])
+      [options setValue:[self delayLoad] forKey:@"delayLoad"];
```

Comment:
Given that the Taplytics SDK is initialized asynchronously by the Segment SDK - does this setting actually work as mentioned when being used via Segment?